spec/workers/concerns/dump_mailer_spec.rb

```diff
@@ -3,7 +3,7 @@
 describe DumpMailer do
   let(:users) { create_list(:user, 2) }
   let(:owner) { users.sample }
-  let(:resource) { Project.new(id: 1, owner: owner) }
+  let(:resource) { create(:project, owner: owner) }
```

Comment:
This shouldn't need to be forced through a DB save to ensure the dump mailer specs work.

Ideally we don't pay the costs of creating factories and the associated records as they slow down the database. This is one of the reason the test suite is so slow on this code base. 

little changes like this add up over time so it's best to avoid this type of change.

Reply:
I think that's fair.  Let me fix this test fix. I think we can get away with stubbing the resource and then have `communication_emails` return what we expect. 

For context, the two tests failing for dump mailer were the tests expecting to send an email to owner, which is set from the project's `communication_emails`.